src/main.py

```diff
@@ -0,0 +1,23 @@
+from fastapi import FastAPI
+import uvicorn
+
+from src.core.configs import Settings
+from src.services.routers.version import router as version_router
+
+app = FastAPI()
+
+# Include the version router
+app.include_router(version_router, prefix="/api")
+
+
+def main():
+    """
+    Main method represents entry point for start application.
+    :return:
```

Comment:
@vasylboliuk according to PEP8 there should be an empty line between first line and arguments list

Reply:
Done